my-megaparsec-project/src/Main.hs

```diff
@@ -0,0 +1,116 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
This is a good start. It would be nice if you took the time to parse the whole expression language. If you decide to fix the submission, please also split your code into separate modules with related functionality.